src/cache.js

```diff
@@ -74,5 +74,17 @@ function key (config) {
   return cacheKey
 }
 
-export { read, write, key }
-export default { read, write, key }
+function invalidate (config = {}) {
+  if (isFunction(config.invalidate)) return config.invalidate
+
+  return async (cfg, req) => {
```

Comment:
Maybe we could extract this function and name it `defaultInvalidate`. It does not use any closures so it does not need to be defined inside the `invalidate` function.

Something like
```js
function invalidate (config = {}) {
  if (isFunction(config.invalidate)) return config.invalidate

  return defaultInvalidate
}

async function defaultInvalidate (config, req) {
  // ...
}
```

Reply:
Ah, yes. That was just a case of lazy programming, should we extract the invalidate related code into a seperate file like you do for other similar features? 

I don't mind, it's not a lot of work to refactor it.

Reply:
I was thinking about it but came to the conclusion that it goes well in the `cache.js` file 😄

Reply:
Fixed